.github/workflows/push.yml

```diff
@@ -80,3 +83,6 @@ jobs:
             -d zend_extension=opcache.so
             -d opcache.protect_memory=1
             -d opcache.jit_buffer_size=16M
+      - name: Assert generated files are up to date
+        if: false # TODO scripts/dev/genfiles needs Bison 3.0
```

Comment:
if someone knows how to fix it, please advise, otherwise let's merge it with this comment

Reply:
I wonder if we actually care about new files at all, or if it's sufficient to just check for modification of tracked files.